### PR TITLE
Add target level option for lint and unittests

### PR DESCRIPTION
**Pull request by WhiteJaeger**

### CODE REVIEW

**Review by dmitry-uraev — COMMENTED** [review]
it seems to me that  check works/ is it?

**Comment by WhiteJaeger** [comment]
> it seems to me that  check works/ is it?

Yes, it does

**Comment by WhiteJaeger** [comment]
Since it is postponed till the second lab - i will do the unittest level here as well.

File: config/run-lint.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export PYTHONPATH="$(pwd)/lab_1:$(pwd)/lab_2:$(pwd)/lab_3:$(pwd)/lab_4:$(pwd):${PYTHONPATH}"
+
+echo ''
```
Comment (demid5111) [inline config/run-lint.sh]: \n

File: config/lab_1/target_tests_6.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+tokenize_test.py
```
Comment (demid5111) [inline config/lab_1/target_tests_6.txt:1]: where is for mark 4?

**Pull request merged**